generation/spade_ldm/spade_ldm_brats.ipynb

```diff
@@ -0,0 +1,1239 @@
UWC1ZXLxDI/vYxOus4/ixtdba3HCspYX72UkzXbjI4ibHGg7yessbHN+3djxj7Mr1F7vi5LJq1SpjtQiZ2E1/EQbiXtAao7wmvmoJYMpDbm9mqQPF8r4IYy1g5CGV48n6gEmNJoV0dFBVC5NUKmWEgBCLBHulXYsIKBGIyWSSgYEBo7mCJehisZgRDIFAwKRY67XJeuYSpnLH5PN5br75ZlasWGFiLIFAwFi0QAHhyPAsLTTlfAiJwISbUacDw4TLUawAuRd1fZUkX0BhjZUoIuIW09vZs9904F/cWlIYKoWy2WwWv99PRUUF1dXVZpCdEJnUYGlLStYuxxT3rxCe1+tlZGSkoPfZ3CcWwPf73Pw7z+Iiyxb24l+a4C1aCBElTIQOrue1Y1uLBspjXbXs4U48q9L8S77FUjo5HWlhf/VGrrcVXLbSManGpJFeWungF+47LrzG3fADHmTXfTv41fLvXnLc5CVu5bvxXyX2eK3VqiWFRSxtsOSJw3yWbxCwxZYihHmKu3n2zHaIqPq3LLxxYhOJpX7Wc4BtPGdcfK10cJy1l7S2d4MrTi7a5ywPnAgFIQMJytvdF3atSwdMdWGaPWipf7RlYffBayE9VSGe/K8zvey+cNFCRSuVtfv9foLBYIFrS9YizQaTyWTBOnK5HIlEglgsZshVuwaFoOcauWjYLU6JH9kz6iSYL7ESfd/oKn3t4pJttCC210nB5BidHF+vwx4vkesnri9t/ch6ZC1CgDrZQFy4uh+eJDKI1SH3kFgqOp1eu2RlX/Jc2DMt7SSpodc5Z7AVtrDXZEG5yJEkwE3s4wYO4iLLa+5bIDv194oRIirmT8pBR7wVf3mSXPYyPku7IfZyLU//t7tpoptGei+axRYlxFnq2cUOYo/UTsx/iY3/3AX3s7NonClCFSdYVkgsgpSD44fXEl0VYiknCBMhSYAIVe/1W04LV5xc+vv7SSQSBSSiYxlao9LbQGFmjrZ85AGVfdknRcrMc/2QardHMQLSpCbQGWH2Nh9yHPGdi7CQtUrsxu4Os9fD6C7MOjMoFosZQhSBpgWmTpGdixASGR0d5ciRI9xwww00NDQUjCywKyFA0esk50mPgJZzrK+LpBprotDKh+5mLdD3iKxbPivXRq6j3M/iZhMrRawQuV+lH564PPXYZlFONEno7yvfRxOLuMQymYzpoSbPgnxHfQ7nJHbv469+/kf80Qf/Aznc7GULvcznpgsMzbLjzeE17KrcYQZ8xfrD7Gcjsf6w2WaAao7RQj1nVRJAiH6qJ4T9xRpkpKDnG8v5v7d9lY+0PMq9PFGUYBIE6GQpr7CJnx77iNWtWLomZ7GsloWw6HePFsROBGm8DFB90UadfWcaeX7BNnK46KKZ/fGNF/kClwdXnFz6+vp44403KC8vp6qqCo/HY9rLy0MqcQ4hDXlA9UMumqA8gCLYk8lkQbWyTknVJCRZY1BYV6D961BY8a0fYNGixaU1PDxs3A/6s6JdlpeXm2JOOZb42sUtAhQIN9FA7dlsIowSiQSpVIozZ87Q19fHXMGFNGWPx0NnZyeHDx+msbGR+vp6E4eQz0Jh2rG91kVeF3LRioUIYx2HkGQIrSxICyAZtW23UEVAw8TIBLFYZBvZZy5njUwYGhoytTeSbCDkARAKhQoKiHUTSk148t3kXpPvKedVsubEGh4YGGBkZKRAKdMKlP26zA3S+QnctYkvt/8nate+zfkzdQTDUY6Ut5IkQBeLL7qHVM88nkjdO/FCXxmx52sLKud/EbuD7k1NrOOAaYDZTRN7uQWOYgXXV4IMoCyKLFY84xD86J5PcW5HvZkKuYJj3MBBEgR4gg/zg10PWi6wHixiiah9xICH4MP8uMBq6aaJXhpJEOAlbuX86fnWG7IPsIjJx3jRqY9fsIVjDS1EIyFGX66wEgQOAUyMgL7cmJFssWeffZZ8Pm/mdNh7OInwFkGgg6TyQMhDqUlF4hzid9YZYXqSoMR1tB9f+9DtsRVZl921Jppof38/PT09DA4OmlksohGDNcZZCw2xRlKpFCMjI6RS1h0gGWoSE5L16FTtZDJpvns6nSYajXL8+HFOnDgxE5fuskATy1SB/eeee86MvtZuP50OrGNm+prJdk6n01gJ0WiUWCxm3I5yfaqqqowA1ym74raSxAB70F+205X6OsVZrAt5v7+/nzNnzhiiFIslk8ngdrupqakxWYRyDInfiYIlhGRP5JB7XL6/JKpIQkhfXx/Hjx/nyJEjk86//TrMDWIZR/bv4Yu/xfmvWXUisZiPH9fci9eXJhELmNb0F8JY/0TXY/qweoMdUBs8BD3B5fTWNPJ8cBsAiVjA6pYcwyKYIBcmF0EMeBJ+ceAOfjFu7az+nVdppYMIYYtY7h/frnl8n4eYiButtKyW9bZ+ae208UT8w2SzLlL9VRPf+wATvcu+hdW65vNYFfkRH4ORBdZ7zwN/C6T2QbGpmpcJM0Iuw8PDdHR0EI/HWbVqFaFQqKCKWrd+kUC7XYsXwtCtxkUzHRsbIxqNkk6njcUj1o8E7MUtARAMBk0mkY7lFMskk/9Fe4xEIrzzzjv09vYyNDSE0znROLOsrIx58+YZYtGCL51OE4vFiEajBe3XRZsVkpNsJkl8yOVy5jsMDQ1x8uRJTp06ZboIzGXomNrZs2c5fPiwuW7BYJCmpibTcFIrAbq3ll3Q28+1WIh+v5/h4WFTpyJEJG4psUbF/SiC3544oGN92iUrpOF0Ok2yhriEpR9cIpFgdHTUpJoHg8GCGhkhELtLdHR0lJGREbO2eDxuOgPIeRQ32PDwMN3d3Zw5c4ZIJFJAhnOKSIriLOwegc+P9+RaCKP3VTBapCXMBSGWxR4sgRxR5+V5B7hhjHJilJvjsFl9vouJ18MXOZYQ0jgO/fxG/uCDX+H06aXwdSA2fuw+h2W1ZMFqie+Ah+Buni5wqXWyjHY2EuuqLTxOFxZpCLlE8hBxwJNMLsDfDaR
```

Comment:
```suggestion
    "        print(\"Epoch %d / %d\" %(epoch, max_epochs))\n",
```
Slight mis-merging issue. I would fix it locally and not here since Github doesn't seem to be showing the change correctly.